src/utils/activity-types.ts

```diff
@@ -0,0 +1,41 @@
+export enum AuditType {
+  LOGIN = 'login',
+  LOGOUT = 'logout',
+
+  CREATE_USER = 'create_user',
+  DELETE_USER = 'delete_user',
+  EDIT_USER = 'edit_user',
+
+  CREATE_PROJECT = 'create_project',
+  DELETE_PROJECT = 'delete_project',
+  EDIT_PROJECT = 'edit_project',
+
+  CREATE_BATCH = 'create_batch',
+  DELETE_BATCH = 'delete_batch',
+  EDIT_BATCH = 'edit_batch',
+
+  CREATE_ITEM = 'create_item',
+  DELETE_ITEM = 'delete_item',
+  EDIT_ITEM = 'edit_item'
+}
+
```

Comment:
Hey @TahaKhanAbdalli  - I'm certainly not the kind of guy who is afraid of `reduce` functions, but... ;-)

Can we handle this through a `map` instead?